docs/source/guides/distributed_training.mdx

```diff
@@ -0,0 +1,193 @@
+
+# To enable ZeRO-1, set the `zero_1` argument to `True` in the training arguments.
+training_args = NeuronTrainingArguments(
+    ...,
+    zero_1=True,
+)
+
+trainer = NeuronTrainer(
+    model,
+    args=training_args,
+    train_dataset=train_dataset,
+    eval_dataset=eval_dataset,
+)
+
+trainer.train()
+```
+
+<Tip>
+Since the [example scripts](https://github.com/huggingface/optimum-neuron/tree/main/examples) use the [`NeuronTrainer`], you can enable ZeRO-1 when using them by add the `--zero_1` flag
+to your command line.
+
+For example:
+
+```bash
+torchrun --nproc_per_node=2 examples/language-modeling/run_clm.py \
+    --model_name_or_path TinyLlama/TinyLlama-1.1B-Chat-v0.6 \
+    --dataset_name wikitext \
+    --dataset_config_name wikitext-2-raw-v1 \
+    --do_train \
+    --per_device_train_batch_size 1 \
+    --block_size 1024 \
+    --bf16 \
+    --zero_1 \
+    --output_dir my_training/
+
+```
+</Tip>
+
+### Via the `NeuronAccelerator`
+
+There is a little bit more work to do when not using the `NeuronTrainer`:
+  
+  1. (Optional) Wrap the optimizer class to make it lazy. When ZeRO-1 is enabled the original optimizer is overridden to use a sharded version of it so it is possible to load original optimizer lazily so that the optimizer state is not materialized until it is actually sharded.
+
+```python
+from torch.optim import AdamW
+from optimum.neuron.distributed import make_optimizer_constructor_lazy
+
+lazy_adamw = make_optimizer_constructor_lazy(AdamW)
+```
+
+  2. Set the `zero_1` argument to `True` when instantiating the `NeuronAccelerator`.
+
+```python
+accelerator = NeuronAccelerator(
+    ...
+    zero_1=True,
+)
+```
+
+
+## How to enable Tensor Parallelism?
+
+Just as for ZeRO-1, it is possible to use Tensor Parallelism both with the [`NeuronTrainer`] and the [`NeuronAccelerator`].
+
+When doing Tensor Parallelism, you have different settings:
+  1. The `tensor_parallel_size`. Ideally it should be smallest value for which the model fits. 
+  2. Whether or not sequence parallelism should be enabled. [Sequence parallelism](https://arxiv.org/pdf/2205.05198.pdf) shards the activations on the sequence axis outside of the tensor parallel regions.
+  It is useful because it saves memory by sharding the activations.
+  3. Whether or not parallelization of the embedding (and thus the LM head for decoder / seq2seq models) should be done. **It is not supported yet**.
+
+On top of that, it is very important to make sure that the original model is loaded in an efficient manner: the training script is going to be called by `torchrun`, which will dispatch it to workers. If each worker (there are 32 of them in  a `trn1.32xlarge` instance) loads the full model weights, it can take a lot of time and go out-of-memory really fast.
+
+`optimum-neuron` provides a context-manager [`distributed.lazy_load_for_parallelism`] that loads the model lazily to prevent that, only the sharded model will be materialized in each worker.
```

Comment:
```suggestion
`optimum-neuron` provides a context-manager [`distributed.lazy_load_for_parallelism`] that loads the model lazily to prevent that, only the parameters for the corresponding model shard will be materialized in each worker.
```